### PR TITLE
Solve security alerts

### PR DESCRIPTION
**Pull request by Shine226**
urllib3, requests, django, notebook

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 alabaster==0.7.10
+appnope==0.1.0
```
Comment (brownsarahm) [inline requirements.txt]: what's this one?

File: requirements.txt [hunk 2]
```diff
@@ -43,6 +44,7 @@ pexpect==4.4.0
 pickleshare==0.7.4
 Pillow==5.0.0
 ply==3.11
+prometheus-client==0.5.0
```
Comment (brownsarahm) [inline requirements.txt]: hat's this

Comment (Shine226) [reply requirements.txt]: When I installed those four libs, prometheus-client was also automately added.

File: requirements.txt (hunk 1 above)
Comment (Shine226) [reply requirements.txt]: When I installed those four libs, appnote was automately added.

**Pull request merged**